WombatProject/Services/GoogleBooksAPIService.cs

```diff
@@ -8,88 +8,43 @@
 using System.Net.Http;
 using System.Threading.Tasks;
 using WombatLibrarianApi.Models;
+using WombatLibrarianApi.Settings;
 
 namespace WombatLibrarianApi.Services
 {
     public class GoogleBooksAPIService : IBookAPIService
     {
-        private readonly IConfiguration _configuration;
-        public WombatBooksContext Context { get; }
-        public List<Book> AuthorBookItems { get; set; } = new List<Book>();
-        public List<Book> SearchResults { get; set; } = new List<Book>();
+        protected readonly GoogleApiSettings _googleApiSettings;
+        public List<Book> AuthorBookItems { get; } = new List<Book>();
+        public List<Book> SearchResults { get; } = new List<Book>();
 
-        public GoogleBooksAPIService(IConfiguration configuration, WombatBooksContext context)
+        public GoogleBooksAPIService(IOptions<GoogleApiSettings> settings)
         {
-            _configuration = configuration;
-            Context = context;
+            this._googleApiSettings = settings.Value;
         }
 
         public async Task GetSearchResults(string searchTerm)
```

Comment:
Method name should be: GetSearchResultsAsync

Reply:
Good point, thanks!